go.mod

```diff
@@ -90,7 +90,7 @@ require (
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
 	github.com/rancher/eks-operator v1.0.6
-	github.com/rancher/gke-operator v1.0.1-rc14
+	github.com/rancher/gke-operator v1.0.1
```

Comment:
The commit looks weird now, and this is already done: https://github.com/rancher/rancher/blob/release/v2.5/go.mod#L93 
Need to rebase I'm guessing